scripts/dts/edtlib.py

```diff
@@ -550,6 +550,14 @@ def _init_prop(self, name, options):
             # 'category: optional' property that wasn't there
             return
 
+        # Handle enum checking
+        enum = options.get("enum")
+        if enum:
+            if type(enum) is not list: enum = [enum]
```

Comment:
I'd prefer to require it to be a list instead, and error out otherwise.

I think dtschema uses `const:` for this. Maybe we could add that instead?

Reply:
From the example-schema.yaml in dtschema:

```
  # Property names starting with '#' must be quoted
  '#interrupt-cells':
    # A simple case where the value must always be '2'.
    # The core schema handles that this must be a single integer.
    const: 2
```

Reply:
(Just discovered that dt-schema supports `default:` by the way. See https://github.com/zephyrproject-rtos/zephyr/pull/17928.)